### DISCUSSION TITLE
alist  报错  ServeHTTP error

### DISCUSSION THREAD

**Discussion by Eric8118** [AlistGo/alist#5444] [General]
🔧[33mWARN🔧[0m[2023-10-29 02:46:40] ServeHttp error. err: write tcp 192.168.10.2:5244->192.168.10.21:55600: write: broken pipe  
🔧[33mWARN🔧[0m[2023-10-29 02:46:40] Maybe size incorrect or reader not giving correct/full data, or connection closed before finish. written bytes: 3691420796 ,sendSize:22667657313,  
🔧[33mWARN🔧[0m[2023-10-29 02:46:40] ServeHttp error. err: write tcp 192.168.10.2:5244->192.168.10.21:55590: write: connection reset by peer  
ALIST运行，日志报以上警告，有时会导致alist未运行，智能重启软路由，求大神解决办法

**Comment by JimiZhou** [comment]
我有同样的问题，在挂载的夸克网盘发送到aria2下载时必现，群晖套件版alist版本3.29.1：

`[33mWARN[0m[2023-12-05 16:31:28] ServeHttp error. err: write tcp 192.168.31.5:5244->192.168.31.5:45768: write: broken pipe  
[33mWARN[0m[2023-12-05 16:31:30] Maybe size incorrect or reader not giving correct/full data, or connection closed before finish. written bytes: 3683 ,sendSize:113246208,  
[GIN] 2023/12/05 - 16:31:28 | 206 |         1m39s |    192.168.31.5 | GET      "/d/夸克网盘/来自：分享/泰坦尼克/时长3：47：00版/4K.120帧.泰坦尼克号.mkv?sign=MoSGUV2Kcdd-8-zhqUU5uqD4uszW50R-tk6giciFrnA=:0"
[GIN] 2023/12/05 - 16:31:30 | 206 | 20.493083081s |    192.168.31.5 | GET      "/d/夸克网盘/来自：分享/泰坦尼克/时长3：47：00版/4K.120帧.泰坦尼克号.mkv?sign=MoSGUV2Kcdd-8-zhqUU5uqD4uszW50R-tk6giciFrnA=:0"
[33mWARN[0m[2023-12-05 16:31:35] ServeHttp error. err: write tcp 192.168.31.5:5244->192.168.31.5:47018: write: broken pipe  
[33mWARN[0m[2023-12-05 16:31:35] Maybe size incorrect or reader not giving correct/full data, or connection closed before finish. written bytes: 3683 ,sendSize:113246208,  
[GIN] 2023/12/05 - 16:31:35 | 206 | 15.259040389s |    192.168.31.5 | GET      "/d/夸克网盘/来自：分享/泰坦尼克/时长3：47：00版/4K.120帧.泰坦尼克号.mkv?sign=MoSGUV2Kcdd-8-zhqUU5uqD4uszW50R-tk6giciFrnA=:0"
[GIN] 2023/12/05 - 16:31:39 | 206 |         1m32s |    192.168.31.5 | GET      "/d/夸克网盘/来自：分享/泰坦尼克/时长3：47：00版/4K.120帧.泰坦尼克号.mkv?sign=MoSGUV2Kcdd-8-zhqUU5uqD4uszW50R-tk6giciFrnA=:0"
[33mWARN[0m[2023-12-05 16:31:52] ServeHttp error. err: write tcp 192.168.31.5:5244->192.168.31.5:45804: write: broken pipe  
[33mWARN[0m[2023-12-05 16:31:53] Maybe size incorrect or reader not giving correct/full data, or connection closed before finish. written bytes: 3683 ,sendSize:109051904,  
[GIN] 2023/12/05 - 16:31:53 | 206 | 12.806646574s |    192.168.31.5 | GET      "/d/夸克网盘/来自：分享/泰坦尼克/时长3：47：00版/4K.120帧.泰坦尼克号.mkv?sign=MoSGUV2Kcdd-8-zhqUU5uqD4uszW50R-tk6giciFrnA=:0"`

下载到一半时alist整个服务挂掉，需要在群晖套件中心重启才能恢复

**Comment by lilii85225** [comment]
解决了吗。我也遇到了，重新获取cookis也不行，也是这样提示，怎样解决
